cmake/SuperBuild.cmake

```diff
@@ -101,19 +101,19 @@ if(UAGENT_FAST_PROFILE)
             )
     endif()
 
-    # Fast RTPS.
-    unset(fastrtps_DIR CACHE)
-    find_package(fastrtps ${_fastrtps_version} EXACT QUIET)
-    if(NOT fastrtps_FOUND)
```

Comment:
I'm not sure it is going to work. Fast-DDS project name is still `fastrtps` so this find package has no effect.

Reply:
Fixed